src/DIPS.Xamarin.UI/Controls/Toast/ToastLayout.cs

```diff
@@ -0,0 +1,79 @@
+using Xamarin.Forms;
+
+namespace DIPS.Xamarin.UI.Controls.Toast
+{
+    /// <summary>
+    ///     Set layout options for the Toast control
+    /// </summary>
+    public class ToastLayout
+    {
+        /// <summary>
+        ///     Gets or sets the color which will fill the background of the Toast
+        ///     <remarks>Default value is <see cref="Color.Black" /></remarks>
+        /// </summary>
+        public Color BackgroundColor { get; set; } = Color.Black;
+
+        /// <summary>
+        ///     Gets or sets the corner radius of the Toast
+        ///     <remarks>Default value is 8</remarks>
+        /// </summary>
+        public float CornerRadius { get; set; } = 8;
+
+        /// <summary>
+        ///     Gets or sets the font family to which the font for the Toast belongs
+        /// </summary>
+        public string FontFamily { get; set; }
```

Comment:
Document default value?

Reply:
I don't agree, the default value is platform specific and I don't see any documentation of this from the Xam.Forms team: https://github.com/xamarin/Xamarin.Forms/blob/8f5090f6062002609051a5c0b17ca16af0989c15/Xamarin.Forms.Core/Label.cs#L200